src/components/Grid/Cell.tsx

```diff
@@ -185,13 +185,11 @@ export default class Cell extends React.Component<Props> {
 
   handleClick: React.MouseEventHandler<HTMLDivElement> = (e) => {
     e.preventDefault?.();
-    e.stopPropagation?.();
```

Comment:
The stopPropagation here is removed so that the click can also focus the input element in GridControls when the event propagates up to that element.